app/assets/stylesheets/content/_work_packages_table.lsg

```diff
@@ -3,9 +3,9 @@
 ## with work packages
 
 ```
-<div class="work-packages-table--container">
-  <div class="work-packages-table--results-container">
-    <table class="workpackages-table">
+<div class="generic-table--container work-package-table--container" interactive-table>
```

Comment:
I think we are not there yet, when it comes to being able to remove the work packages table from the LSG. 

This table still has some unique capabilities (although they do not work in the LSG): column menu, row menu, ...

But until we are able to have the same capabilities for all the tables, I would leave it there.